avm/res/search/search-service/main.bicep

```diff
@@ -530,3 +557,20 @@ type diagnosticSettingType = {
   @description('Optional. The full ARM resource ID of the Marketplace resource to which you would like to send Diagnostic Logs.')
   marketplacePartnerResourceId: string?
 }[]?
+
+type secretsExportConfigurationType = {
+  @description('Required. The key vault name where to store the API Admin keys generated by the modules.')
+  keyVaultResourceId: string
+
+  @description('Optional. The primaryAdminKey secret name to create.')
+  primaryAdminKey: string?
+
+  @description('Optional. The secondaryAdminKey secret name to create.')
+  secondaryAdminKey: string?
```

Comment:
```suggestion
  primaryAdminKeyName: string?

  @description('Optional. The secondaryAdminKey secret name to create.')
  secondaryAdminKeyName: string?
```

Note: Needs to be updated in other places too.

Reply:
Everything else looks great. Thanks @Agazoth

Reply:
@AlexanderSehr  I need help. I have implemented the changes and I have run Set-AVMModule with -Recurse. Everythins should be up to date, but I keep getting this error:
![{1A039F28-642C-468D-95B4-447D1A09DFAD}](https://github.com/user-attachments/assets/4d79bea0-9e21-4c0d-b32c-846ab2172402)

Unfortunately, the info does not make me any wiser. I have rerun the Set-AVMModule cmdlet sevderal times, but there are no changes, when running it on my machine. I have checked for wierd sorting issues, but none of the parameters in the nested module have initial characters that would trigger danish sorting.

The only thing changed in the readme is this:
![{9713998C-01A5-4C85-8FD8-0104DB83858F}](https://github.com/user-attachments/assets/8bc2d6a6-f6d5-4307-b848-013fcf36ed02)


There seems to be missing a / in the new one and I am not sure Microsoft. part should be removed. This happened after running Set-AVMModule - I did not change the file manually.

Reply:
Well - that was exactly it - it even said so in the log.

I have updated the readme.mds manually.

It is really strange. Running Update-AVMModule does nothing to the ReadMes, but running Set-ModuleReadMe implements the error:
![{52572748-E79A-4229-80B1-1C70DA887261}](https://github.com/user-attachments/assets/dcdef3d5-2fc2-41d1-9b99-ab8919fd61d4)

Reply:
After 9 failed attempts  finally succeded in getting the REadMes in a state that the pipeline accepts. Only the pe step remains. It has been running for 52 minutes now and has been removing resources for 20+ minutes. Last time I ran the pe step successfully, it took 1 hour and 56 minutes, so I expect it to succeed in about an hour 😄